### PR TITLE
Removes kudzu grief

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
Kudzu is unplantable, admens get a button to remove it for when the random event one gets too funky i suppose

:cl: McDonald072
rscdel: Kudzu has been removed until fuhrer notice.
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
1.Need admin on. 2. Spams admin with seed plant. 3 Won't prevent lag that happens MOMENTALLY. 
4. Won't fix lag - Xantam rebooted it from host box. 5. Deletes all hard work of antag if lag HAPPENS.
5.Shitty massproducable griefweapon you cannot fight ingame by any means.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
1. Yeah, that's the point. You want an admin to supervise potentially server crashing stuff.
2. You call one line of text spamming?
3. Not quite sure what you mean here?
4. There's a button right there to delete the lag before it gets out of control
5. Good for hijacking eh?

**Comment by ShadowDeath6** [comment]
just remove the ability to plant them, it's cancerous and no admin is going to approve it unless they want to ruin the round

or better yet, pr this but have the only thing be the 'delete all vines' button

also you can just spam the fuck out of z and it will annihilate the admins with sound

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -1044,10 +1045,29 @@ var/list/admin_verbs_hideable = list(
...
+	set desc = "Performs advanced magic to delete all vines in existence."
 
+	if(!holder)
+		return
 
-
-
+	SSobj.can_fire = 0 //So vines can't create more of themselves in their dying breath
+	CHECK_TICK
+	src << "<span class='notice'><b>Erasing vines, this might lag for a bit...</b></span>"
```
Comment (ShadowDeath6) [inline code/modules/admin/admin_verbs.dm]: does this correctly display

also I would just have normal notice, not bolt

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
If I remove planting kudzu what is the point of delete all vines

**Comment by ShadowDeath6** [comment]
then don't remove planting kudzu if you're adding the new button

**Comment by Ratvar** [comment]
3,4-2.0:Kudzu gargantuan lag happens in seconds without warning. Massdelete won't prevent the problem.
5: Too easy for hjacking cuz unstoppable.
6: Buffed kudzu sometimes grows through nullspace

**Comment by Partheo** [comment]
Not a fan of this PR. The remove all vines button is good, but pinging admins to ask them "hey please approve this grief" is not the right direction to go in. If Kudzu is such a grief problem that is deserves a fix(which it does) then port nerfs and tweak it. We're not going to ask admins every time someone wants to detonate a maxcap, are we?

**Comment by Jc1469** [comment]
I'll quote Xantam.

[6:12 PM] Xantam: If it is never made, it'll never get fixed. I won't accept a fix before removal.

And this is neither a fix nor removal.  It's a stepping stone which turns lag into admin-enabled lag.  And the lag that was happening was lag of verbs not working. So this verb wont really help with that if it ever gets to that point again.

**Comment by ShadowDeath6** [comment]
>I won't accept a fix before removal

nice meme

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Guess this now is the kudzu removal PR @05rhardy wanted

**Comment by X-TheDark** [comment]
If it's a problem and there's no immediate `IMPROVE` suggestion, then I'll just have to go with this PR to `REMOVE, DON'T` for now.

**Comment by Xantam-13** [comment]
This removes Kudzu/Vines from entering the game via players right (now you've amended it)?

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Yep, only random event kudzu remains

**Comment by X-TheDark** [comment]
Priority merge of this has been requested by Xantam.

**Pull request merged**